ansible/roles/test/files/tools/loganalyzer/loganalyzer_common_ignore.txt

```diff
@@ -183,3 +183,6 @@ r, ".* ERR pmon#sensord: Error getting sensor data: dps.*"
 
 # https://msazure.visualstudio.com/One/_workitems/edit/24444744/
 r, ".* ERR syncd\d*#syncd.*SAI_API_UNSPECIFIED:sai_bulk_object_get_stats.*"
+
+# https://msazure.visualstudio.com/One/_workitems/edit/25018599
+r, ".* ERROR: Failed to parse lldp age.*"
```

Comment:
Seems these error messages in case were a result of the time reset which was caused by the script and should be an expected behavior.
What about if these error messages occur in a standard environment, would they require any additional action to handle or check?

Reply:
normally, if parse age failed, throw exception, this age format is invalid. it will not fill age fields. Not impact lldp-syncd process.